pkg/rancher-desktop/assets/scripts/wsl-exec

```diff
@@ -9,6 +9,14 @@
 
 set -o errexit -o nounset
 
+# We may have WSLENV set to override PATH; however, that causes the default entries to be missing.
+IFS=':' read -ra dirs <<< /usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
```

Comment:
The shebang says `/bin/sh`; this seems to contain bashisms?

Reply:
Ah, yes, good catch

Reply:
New variant tested like this:

```
/mnt/c/Users/SUSE # cat addpath.sh
str="/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:"
while [ -n "$str" ]; do
  dir=${str%%:*}
  if ! [[ ":${PATH}:" =~ :${dir}: ]]; then
    export PATH="${PATH%:}:${dir}"
  fi
  str=${str#*:}
done
/mnt/c/Users/SUSE # PATH=/bin
/mnt/c/Users/SUSE # echo $PATH
/bin
/mnt/c/Users/SUSE # . addpath.sh
/mnt/c/Users/SUSE # echo $PATH
/bin:/usr/local/sbin:/usr/local/bin:/sbin:/usr/sbin:/usr/bin
```